requirements.txt

```diff
@@ -1,4 +1,4 @@
-dwave-ocean-sdk==5.0.0
+dwave-ocean-sdk>=6.3.0
```

Comment:
FWIW, since we require `dwave-inspector>=0.4.0` in latest prod IDE, it's sufficient to require `dwave-ocean-sdk>=6.2.0`.